<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSelect</class>
 <widget class="QWidget" name="ImageSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageSelect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addImage">
       <property name="text">
        <string>添加影像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addFolder">
       <property name="text">
        <string>添加文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearImage">
       <property name="text">
        <string>清空影像</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
